src/linagora.esn.contact/app/addressbook/settings/delegation/contact-addressbook-settings-delegation.controller.js

```diff
@@ -1,7 +1,7 @@
 const _ = require('lodash');
 
-require('../../../sharing/sharee/contact-sharee.service.js');
-require('../../../sharing/constants.js');
+//require('../../../sharing/sharee/contact-sharee.service.js');
+//require('../../../sharing/constants.js');
```

Comment:
comments? Remove if they are not required.

Reply:
oh -_- , i forgot it ,there is a huge number of files :D